lib/filewatch/observing_base.rb

```diff
@@ -0,0 +1,68 @@
+# encoding: utf-8
+require_relative 'bootstrap' unless defined?(FileWatch)
```

Comment:
why is this required?

Reply:
IMO because I wanted the file input to not need to know about bootstrap.rb. So the entry point is either ObservingTail or ObservingRead.

Reply:
but can't the require order be done in a way that we don't need this kind of conditional require? in my experience these become hard to maintain.

Reply:
I can remove the `unless`.